src/screens/pos-actions/pos-action-redelegate/redelegate-enter-amount/redelegate-enter-amount.tsx

```diff
@@ -83,15 +83,29 @@ export class RedelegateEnterAmountComponent extends React.Component<
         });
 
         this.state = {
-            amount: '',
-            insufficientFunds: false,
-            feeOptions: undefined,
-            insufficientFundsFees: false
+            amount: undefined
         };
     }
 
     public componentDidMount() {
         this.props.navigation.setParams({ actionText: this.props.actionText });
+
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+        const tokenConfig = getTokenConfig(this.props.blockchain, this.props.token.symbol);
+        blockchainInstance
```

Comment:
use async/await

Reply:
The reason i am not using async/await is that the value that is need to get is not mandatory for the screen to display. The balance value can be shown after the entire screen is rendered.